### PR TITLE
Update readmes and contribs

### PR DESCRIPTION
**Pull request by jansenbob**
Update readmes and contribution guides for bach, modelhub and tracker packages

### CODE REVIEW

File: bach/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,4 +1,89 @@
...
+the following command:
+```bash
+make tests-all
+```
+
+## See Also
+* [Pandas](https://github.com/pandas-dev/pandas): the inspiration for the API.
+   Pandas has excellent [documentation](https://pandas.pydata.org/docs/) for its API.
+* [SQL-models](./sql_models/): Sub-project that is used for generating the underlying sql-queries. Can be 
+  found in the [`sql_models`](./sql_models/) package
```
Comment (ivarpruijn) [inline bach/CONTRIBUTING.md:73]:
```suggestion
  found in the [`sql_models`](./sql_models/) package.
```

File: bach/README.md [hunk 2]
```diff
@@ -1,100 +1,39 @@
...
+Bach is a Python-based analytics modeling library with a Pandas-like interface. It translates all operations and models to SQL, running directly on your full dataset. All DataFrames & models can be converted to an SQL statement with just a single command, to use in other tools like dbt or BI.
 
-Bach uses [`sql_models`](./sql_models/) under the hood, which makes it possible to easily build graphs of SQL models and generate SQL for the resulting composite `sql-models`. See [sql_models/README.md](./sql_models/README.md) for more information.
+Bach is part of [Objectiv](https://objectiv.io/), an open-source data collection & modeling platform that helps data teams run product analytics from their notebooks. Bach therefore works with any dataset that embraces Objectiv's [open analytics taxonomy](https://objectiv.io/docs/modeling/open-model-hub/), so you can take pre-built models from [open model hub](https://objectiv.io/docs/taxonomy/) to quickly build deep analyses for a wide range of product & marketing use cases.
```
Comment (ivarpruijn) [inline bach/README.md:5]:
```suggestion
Bach is part of [Objectiv](https://objectiv.io/), an open-source data collection & modeling platform that helps data teams run product analytics from their notebooks. Bach therefore works with any dataset that embraces Objectiv's [open analytics taxonomy](https://objectiv.io/docs/taxonomy/), so you can take pre-built models from the [open model hub](https://objectiv.io/docs/modeling/open-model-hub/) to quickly build deep analyses for a wide range of product & marketing use cases.
```

File: tracker/trackers/angular/README.md [hunk 3]
```diff
@@ -2,8 +2,10 @@
 
 Objectiv tracker for Angular that embraces the open analytics taxonomy. Supports all core tracking methods out of the box, and provides added shorthands for Angular components. Also provides default auto-tracking (ApplicationLoaded) out of the box.
```
Comment (ivarpruijn) [inline tracker/trackers/angular/README.md:3]: Add link to taxonomy here?

File: tracker/trackers/react/README.md [hunk 4]
```diff
@@ -2,8 +2,10 @@
 
 Objectiv tracker for React applications that embraces the open analytics taxonomy. Supports core tracking methods and default auto-tracking (ApplicationLoaded) out of the box.
```
Comment (ivarpruijn) [inline tracker/trackers/react/README.md:3]: Add link to taxonomy here?

File: tracker/trackers/browser/README.md [hunk 5]
```diff
@@ -2,8 +2,10 @@
 
 Objectiv tracker for web applications & websites that embraces the open analytics taxonomy. Supports core tracking methods and default auto-tracking (ApplicationLoaded) out of the box.
```
[the remark this answers is not in the document]
Comment (jansenbob) [reply tracker/trackers/browser/README.md]: done

File: tracker/trackers/angular/README.md (hunk 3 above)
Comment (jansenbob) [reply tracker/trackers/angular/README.md]: done

File: tracker/trackers/react-native/README.md [hunk 6]
```diff
@@ -1,7 +1,9 @@
-# Objectiv React Tracker
+# Objectiv React Native Tracker
 
 Objectiv tracker for React Native applications that embraces the open analytics taxonomy. Supports core tracking methods and default auto-tracking (ApplicationLoaded) out of the box.
```
[the remark this answers is not in the document]
Comment (jansenbob) [reply tracker/trackers/react-native/README.md]: done

File: tracker/trackers/react/README.md (hunk 4 above)
Comment (jansenbob) [reply tracker/trackers/react/README.md]: done

**Comment by ivarpruijn** [comment]
LGTM!

**Pull request merged**